docs/decisions/0001-adding-new-checks.rst

```diff
@@ -0,0 +1,36 @@
+Adding New Checks
+=================
+
+Status
+------
+
+Draft
+
+Context
+-------
+
+When adding new violations to edx-lint, there currently isn't clarity around the following:
+
+* What changes should be considered a breaking change, requiring a major version bump.
+* How to avoid too avoid many breaking changes.
+* When and where to best communicate changes.
+* How to ensure that new violations are being picked up by repositories.
+
+Decisions
+---------
+
+* New checks should be introduced as optional, even if you ultimately want them to be required by default.
+
+  * For rolling out new required-by-default checks, making checks optional at first enables repo owners to fix or amnesty violations on their own timeline before they become required, or batch fixing or amnestying larger batches of checks when they become required.
+  * See `how_tos/0001-adding-new-checks.rst` for details in making this happen.
+
+* Mark new checks that will become required as deprecated as disabled/optional checks.
```

Comment:
I don't understand this sentence.

Reply:
Me neither. 😄 I was rushing on the ADR, trying to balance not repeating too much of the how-to, but also providing a summary of decisions that might change. Once the rest is cleaned-up, I will fix this.  Thanks.